frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -332,7 +332,7 @@ function ReaderRolling:onPageUpdate(new_page)
 end
 
 function ReaderRolling:onResume()
-    self:updateBatteryState()
```

Comment:
Okay, so don't remove this line 'cause it's not redundant according to @Hzj-jie ;-)